src/Entity/Intervention.php

```diff
@@ -104,6 +105,25 @@ public function setScheduledAt(?\DateTimeImmutable $scheduledAt): self
         return $this;
     }
 
+    public function getScheduledAtFormated(): string
+    {
+        if ($this->getScheduledAt()->format('H') > 0) {
+            if ($this->getPartner()) {
```

Comment:
C'est pas plus simple de faire juste :
```
if ($this->getScheduledAt()->format('H') > 0) {
    $timezone = $this->getPartner()?->getTerritory()?->getTimezone() ?? TimezoneProvider::TIMEZONE_EUROPE_PARIS;
    return $this->getScheduledAt()
                ->setTimezone(new \DateTimeZone($timezone))
                ->format('d/m/Y à H:i');
}
return $this->getScheduledAt()->format('d/m/Y');
```
?

Reply:
et pour l'amour de l'art, je propose de  comparer sur `if ($this->getScheduledAt()->format('His') > 0) {`

Reply:
Ok, j'ai repris vos suggestions !